tests/python_tests/samples/test_whisper_speech_recognition.py

```diff
@@ -5,24 +5,36 @@
 import pytest
 import sys
 
-from conftest import SAMPLES_PY_DIR, SAMPLES_CPP_DIR
+from conftest import SAMPLES_PY_DIR, SAMPLES_CPP_DIR, SAMPLES_C_DIR
 from test_utils import run_sample
 
 class TestWhisperSpeechRecognition:
     @pytest.mark.whisper
     @pytest.mark.samples
     @pytest.mark.parametrize("convert_model", ["WhisperTiny"], indirect=True)
     @pytest.mark.parametrize("download_test_content", ["how_are_you_doing_today.wav"], indirect=True)
-    def test_sample_whisper_speech_recognition(self, convert_model, download_test_content):           
+    def test_py_sample_whisper_speech_recognition(self, convert_model, download_test_content):           
         # Run Python sample
         py_script = os.path.join(SAMPLES_PY_DIR, "whisper_speech_recognition/whisper_speech_recognition.py")
         py_command = [sys.executable, py_script, convert_model, download_test_content]
-        py_result = run_sample(py_command)
+        run_sample(py_command)
 
+    @pytest.mark.whisper
+    @pytest.mark.samples
+    @pytest.mark.parametrize("convert_model", ["WhisperTiny"], indirect=True)
+    @pytest.mark.parametrize("download_test_content", ["how_are_you_doing_today.wav"], indirect=True)
+    def test_cpp_sample_whisper_speech_recognition(self, convert_model, download_test_content):
         # Run C++ sample
-        cpp_sample = os.path.join(SAMPLES_CPP_DIR, 'whisper_speech_recognition')
+        cpp_sample = os.path.join(SAMPLES_CPP_DIR, 'whisper_speech_recognition', 'whisper_speech_recognition')
         cpp_command = [cpp_sample, convert_model, download_test_content]
-        cpp_result = run_sample(cpp_command)
+        run_sample(cpp_command)
 
-        # Compare results
-        assert py_result.stdout == cpp_result.stdout, "Python and C++ results should match"
```

Comment:
Please return python and cpp results assert back. Please add C sample results assert to match python and cpp samples.

Reply:
Good catch, reverted and added assertion for C results